src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -1029,6 +1030,10 @@ bool fuse_type_to_constant(const std::shared_ptr<ngraph::Node>& node,
                            const precisions_map& precisions,
                            const std::vector<Input<Node>>& consumers) {
     auto from = node->get_element_type();
+    // Consts marked with disable_constant_folding should be kept in f16 until they reach to the CPU plugin
+    if (from == element::f16 && constant_folding_is_disabled(node))
```

Comment:
Do we really need to check precision is equal f16? Isn't it enough to check `constant_folding_is_disabled(node)` condition?

Reply:
done